addons/crm_iap_mine/static/src/views/generate_leads_views.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<templates>
+    <t t-name="LeadMiningRequest.generate_leads_button" owl="1">
+        <button t-if="isSalesManager" type="button" class="btn btn-secondary o_button_generate_leads" t-on-click="onClickGenerateLead">
+            Generate Leads
+        </button>
+    </t>
+
+    <t t-name="LeadMiningRequestListView.buttons" t-inherit="web.ListView.Buttons" t-inherit-mode="primary" owl="1">
+        <!-- Before the export button -->
+        <xpath expr="//t[@t-if='props.showButtons']//t[3]" position="before">
```

Comment:
Is this targeting the "t" element based on position?
Does not seem super robust, can't we use a class or a name or something?

Reply:
Our options are quite limited unfortunately.. We can either:
 - `<t t-elif="activeActions['create'] and (!model.root.isGrouped or !editable)">` position="after"
 - `<t t-if="nbTotal and !nbSelected and activeActions['exportXlsx'] and isExportEnable and !env.isSmall">` position="before"
Perhaps something like this is more appropriate:
```suggestion
        <xpath expr="//t[contains(@t-if, 'isExportEnable')]" position="before">
```